### PR TITLE
cypress: add tests for tracked comments operation

### PR DESCRIPTION
**Pull request by lpranam**
Change-Id: If1a2bbe7d9d3ec7fa821db91aea1f7b3ed6f2c1e

* Target version: distro/collabora/co-23.05 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by gokaysatir — APPROVED** [review]
Thanks :)

File: cypress_test/integration_tests/desktop/writer/track_changes_spec.js [hunk 1]
```diff
@@ -42,7 +45,35 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Track Changes', function (
 
 	it('Accept All', function () {
 		helper.typeIntoDocument('Hello World');
+		for (var n = 0; n < 2; n++) {
+			cy.cGet('#tb_editbar_item_insertannotation').click();
+			cy.cGet('#annotation-modify-textarea-new').type('some text' + n);
+			cy.cGet('#annotation-save-new').click();
+			// Wait for animation
+			cy.wait(500);
+		}
```
Comment (neilguertincollabora) [inline cypress_test/integration_tests/desktop/writer/track_changes_spec.js:54]: There is a function in desktop_helper called insertMultipleComment that can do this.

File: cypress_test/integration_tests/desktop/writer/track_changes_spec.js [hunk 2]
```diff
@@ -8,8 +8,11 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Track Changes', function (
 	var testFileName;
 
 	beforeEach(function () {
+		cy.viewport(1400, 600);
```
Comment (neilguertincollabora) [inline cypress_test/integration_tests/desktop/writer/track_changes_spec.js:11]: Why are the viewport and zoom changes necessary?

Comment (lpranam) [reply cypress_test/integration_tests/desktop/writer/track_changes_spec.js:11]: To make comments fully visible

File: cypress_test/integration_tests/desktop/writer/track_changes_spec.js (hunk 1 above)
Comment (lpranam) [reply cypress_test/integration_tests/desktop/writer/track_changes_spec.js:54]: Nah, that method uses a different UI (i.e: notebook bar) which also requires some additional UI setup, it was causing more problems and required existing tracked comment tests to change. This was the easy way around.

**Review by neilguertincollabora — APPROVED** [review]

**Pull request merged**